nodejs/pilotUtils.js

```diff
@@ -0,0 +1,351 @@
+// Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+
+// Permission is hereby granted, free of charge, to any person obtaining a copy 
+// of this software and associated documentation files (the "Software"), to 
+// deal in the Software without restriction, including without limitation the 
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or 
+// sell copies of the Software, and to permit persons to whom the Software is 
+// furnished to do so, subject to the following conditions:
+
+// The above copyright notice and this permission notice shall be included in 
+// all copies or substantial portions of the Software.
+
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR 
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE 
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER 
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING 
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
+// DEALINGS IN THE SOFTWARE.
+
+const axios = require('axios');
+// const { exit } = require('process');
+
+// const args = process.argv.slice(2);
+const uuid = '<Your UUID>'; //args[0];
```

Comment:
again these can be set by command parameters args[0], args[1], etc.

Reply:
For this I wasn't sure what to do. I wanted to leave those to show that this could be called from the command line but also you could include the variables in the file and have like a single button push kind of functionality. Would it be more common for something like this to run via cli?

Reply:
If the user experience is actually to alter the script itself withOUT cli args then you should keep it how you have it.  You'll need to make sure this is communicated also in the user documentation :-)  Btw, is Ben W. doing that documentation for you?

Reply:
I've been thinking about your comment here and I have a meeting with Corey, Morgan, Dylan and Dave tomorrow and I want to address this because you're absolutely right that these need to be standardized so I want to bring this up to them and see what they want out of it

Reply:
ok I will be prepared to re-review if changes come out of your upcoming meeting.  Thanks Adam!